Common/include/CConfig.hpp

```diff
@@ -1990,6 +2019,47 @@ class CConfig {
    */
   su2double GetInc_Temperature_Init(void) const { return Inc_Temperature_Init; }
 
+  void SetNSpecies(unsigned short n_species) { this->n_species = n_species; }
+  unsigned short GetNSpecies(void) const { return n_species; }
+
```

Comment:
👍 The logic behind nSpecies, nVar etc surely needs some consolidation throughout the implementation. With that this will probably go away.

The `this->` logic will also go away (as well as the `void`)